src/literals.py

```diff
@@ -74,6 +74,12 @@ class Status(Enum):
         BlockedStatus("failed to enable authentication in etcd"), "ERROR"
     )
     BACKUP_IN_PROGRESS = StatusLevel(MaintenanceStatus("Creating database backup..."), "DEBUG")
+    BACKUP_S3_PARAMETERS_MISSING = StatusLevel(
+        BlockedStatus("S3 parameters are missing - please provide them"), "ERROR"
```

Comment:
I would perhaps rephrase this into a more passive to imply that they are missing but do not need to be provided "directly" to etcd rather to the "sobject storage" integrator, perhaps:
```suggestion
        BlockedStatus("Missing required parameters in the s3 relation."), "ERROR"
```